lib/paper_trail/has_paper_trail.rb

```diff
@@ -269,6 +270,7 @@ def touch_with_version(name = nil)
 
         attributes.each { |column| write_attribute(column, current_time) }
         save!
+        record_update(true) if self.class.paper_trail_options[:on] == []
```

Comment:
@batter Yes, I see that ```save!(:validate => false)``` should be enough, but because my record doesn't change, it seems like active record wonth fire after_update, and hence the version will no be created.
```
Foo.last.touch_with_version
Foo Load (0.4ms)  SELECT  "foos".* FROM "foos"  ORDER BY "foos"."id" DESC LIMIT 1
   (0.1ms)  begin transaction
   (0.1ms)  commit transaction
 => true 
```
But this seems like it will work with the PR from @jaredbeck . Thank you